android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/MapCompat.java

```diff
@@ -13,6 +13,10 @@
   private static final String TAG = "MapCompat";
 
   public static <K, V> V getOrDefault(HashMap<K, V> map, K key, V defaultValue) {
+    if (map == null) {
```

Comment:
I'd prefer this to still throw on `null`. It's invalid input.

If we want to do anything about that, the parent function `com.tavultesoft.kmapro.KeyboardSettingsActivity$2.onItemClick` already has a test for `hashMap == null` and logs that. Just return from that test after L152:

https://github.com/keymanapp/keyman/blob/493bb945b61f0ed0a625a45dd4e8041dcc7e047c/android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/KeyboardSettingsActivity.java#L150-L153

Reply:
We don't actually need this test at all, just let it bork as it was previously!